app/models.py

```diff
@@ -2599,6 +2607,26 @@ def aliases(self) -> [Alias]:
 
         return ret
 
+    @classmethod
+    def create(cls, **kw):
+        # whether to call Session.commit
+        commit = kw.pop("commit", False)
+        flush = kw.pop("flush", False)
+
+        if "email" in kw:
```

Comment:
can we call Base.create() after sanitizing the email that will take care of the commit and flush options?

Reply:
Done